actions/workflows/get_error_data.yaml

```diff
@@ -0,0 +1,84 @@
+---
+version: 1.0
+
+description: Gets the execution tree and the resulting error of a given execution
+
+input:
+  - cmdb_request_item
+  - cmdb_request_item_url
+  - retire
+  - st2_exe_id
+  - sn_update
```

Comment:
See comment above. I think this is probably used to know if the values should be newlines vs line breaks so we should call it something else so and that can be passed in and the user knows what to expect